src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/ServiceConfigItem.java

```diff
@@ -20,6 +31,8 @@
 
     private boolean allowCreatingCaseBeforePaymentsAreProcessed = false;
 
+    private Map<String, String> formTypeToSurnameOcrFieldMappings = new HashMap<>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [If you run in Java5 or newer and have concurrent access, you should use the ConcurrentHashMap implementation](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4498197)